packages/react/src/components/CardLink/CardLink.js

```diff
@@ -32,6 +38,10 @@ CardLink.propTypes = {
    * See [`<Card>`'s README](http://ibmdotcom-react.mybluemix.net/?path=/docs/components-card--link-clickable#props) for full usage details.
    */
   card: PropTypes.shape(Card.propTypes).isRequired,
+  /**
+   * disabled: `true` to disable it.
+   */
+  disabled: PropTypes.bool,
```

Comment:
@greeshmareddyg can you add `defaultProps` and set `disabled` to be `false` by default?